tests/foreman/ui/test_computeresource_gce.py

```diff
@@ -127,9 +97,9 @@ def test_positive_default_end_to_end_with_custom_profile(
         cr_values = session.computeresource.read(new_cr_name)
         assert cr_values['name'] == new_cr_name
         assert (set(cr_values['organizations']['resources']['assigned'])
-                == {module_org.name, new_org.name})
+                == {org.name, new_org.name})
```

Comment:
no need for extra brackets after assert

Reply:
Flake8 was complaining for 'needs statement'. I reformated things.